playbooks/roles/stackdriver/defaults/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+
+stackdriver_distro_codename: "precise"
+stackdriver_repo_url: "http://repo.stackdriver.com/apt"
+stackdriver_pubkey_url: "https://app.stackdriver.com/RPM-GPG-KEY-stackdriver"
+stackdriver_api_key: ""
+stackdriver_detect_gcm: "yes"
```

Comment:
On GCP, the Stackdriver agent can automatically configure itself to talk to Google Cloud Monitoring. On AWS, we'd set this to "no" and use the API key instead.